### PR TITLE
Deploy beta version of PCA on subdomain

### PR DESCRIPTION
**Pull request by davish**
Just added the wildcard domain record to google domains, will `dig` it before deploying.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -60,9 +60,20 @@ workflows:
           tag: "${CIRCLE_SHA1},latest"
           context: docker-publish
           <<: *branch-filters
+      - docker/publish:
+          name: publish-alert
+          requires:
+            - pcp-lint
```
Comment (pawalt) [inline .circleci/config.yml]: jk deleto this

**Comment by pawalt** [comment]
Also can we turn on linting for the alert frontend?

File: k8s/values.yaml [hunk 2]
```diff
@@ -7,14 +7,25 @@ applications:
   - name: celery
     image: pennlabs/penn-courses-backend
     secret: penn-courses
-    cmd: ["celery", "worker", "-A", "PennCourses", "-Q", "alerts,celery", "-linfo"]
+    cmd:
```
Comment (pawalt) [inline k8s/values.yaml]: this is not how yaml works. can't make newlines. Refer to below comment on how to newline this

File: k8s/values.yaml [hunk 3]
```diff
@@ -7,14 +7,25 @@ applications:
...
+      ["celery", "worker", "-A", "PennCourses", "-Q", "alerts,celery", "-linfo"]
     extraEnv:
       - name: DJANGO_SETTINGS_MODULE
         value: PennCourses.settings.production
   - name: beat
     image: pennlabs/penn-courses-backend
     secret: penn-courses
-    cmd: ["celery", "-A", "PennCourses", "beat", "-l", "info", "--scheduler", "django_celery_beat.schedulers:DatabaseScheduler"]
+    cmd:
```
Comment (pawalt) [inline k8s/values.yaml]: again not how yaml works. this needs to be something like
```yaml
cmd:
  - celery
  - -A
...
```

File: .circleci/config.yml [hunk 4]
```diff
@@ -31,6 +32,27 @@ jobs:
...
+      - restore_cache:
+          key: dependency-cache-{{ checksum "frontend/alert/package-lock.json" }}
+      - run:
+          name: Install dependencies
+          command: |
+            cd frontend/alert && npm install
+      - save_cache:
+          key: dependency-cache-{{ checksum "frontend/alert/package-lock.json" }}
+          paths:
+            - frontend/plan/node_modules
```
Comment (ArmaanT) [inline .circleci/config.yml]: Typo here. Should probably be `alert` not plan

**Pull request merged**